site/en/tutorials/keras/text_classification_with_hub.ipynb

```diff
@@ -139,6 +139,7 @@
         "  pass\n",
         "import tensorflow as tf\n",
         "\n",
+        "!pip install tensorflow-hub\n",
```

Comment:
@npanpaliya , this example should pip install both tensorflow-datasets and tensorflow-hub as if they run this docker file first, they may not have either installed.

Reply:
Thanks @wdirons. I've added that now although when I ran this notebook it didn't complain for datasets here, it might have found it installed.

Reply:
Thank you @npanpaliya. If you created a new docker container and tried this notebook first, it would have complained about tensorflow-datasets not being installed, but if you ran the other notebook with your change to install tensorflow-datasets first, it would have already been installed.